common/src/cams/orders.ts

```diff
@@ -25,6 +26,32 @@ export type TransferOrder = CaseSummary & {
   reason?: string;
 };
 
+// TODO: Helper function while we are in transition to remodel "has a".
+export function getCaseSummaryFromTransferOrderYuck(order: TransferOrder) {
```

Comment:
Gotta talk about how and when we are going to refactor the common model so that this and the other Yuck function are no longer necessary.